docs/source/models/risk_correlation/sbp_ldlc_fpg_bmi/index.rst

```diff
@@ -68,6 +68,38 @@ between 1 and 0 with the same correlation between variables.
 The calculations are done `in this notebook <https://github.com/ihmeuw/vivarium_research_nih_us_cvd/blob/main/Correlation_Testing.ipynb>`_. This code can be used to generate propensity 
 values for any number of simulants. 
 
+Joint PAF Calculations
+++++++++++++++++++++++
+
+As the risks are now correlated, we will need adjust the PAF values to account for this. More 
+information on why this is necessary can be found on `Vivarium here <https://vivarium-research.readthedocs.io/en/latest/model_design/vivarium_model_components/risk_factors/risk_correlation/index.html#pafs-for-correlated-risks>`_. 
+
+We will be calculating the joint PAFs in a similar approach to the uncorrelated and 
+unmediated PAF for BMI to heart failure. Therefore, we will initialize a large 
+population once and then save the calculated PAF values to the artifact for future use.
+
+As a proof of concept, the research team has done this calculation in `a notebook <https://github.com/ihmeuw/vivarium_research_nih_us_cvd/blob/main/PAF_interactive.ipynb>`_. However, at the 
+time this workbook was made, the risks in the interactive sim were not correlated 
+and mediation was not included in the model. Therefore, we expect that this calculation 
```

Comment:
This notebook can be an external check on the implementation of the correlation in the sim, too.